### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by michaelhigham**
Add information about tagging.

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Yes, see [API Reference](/api).
...
+The easiest way is to have different [Infra Apps](/monitoring/infrastructure) for each environment, then a [Logs](/logs)/[Monitoring](/monitoring) App for each environment as well. The Infra App of each enviromnet will be linked to the right Logs App via [Logs Discovery](/logs/discovery/intro) and to the right Monitoring App via [Autodiscovery](/monitoring/autodiscovery). If it is not possible to create a seperate App for each environment, then it also possible to use [Custom Tags](/tags/custom-tags) to keep your data separate. The data can then be further isolated with the use of [Saved Views](/guide/saved-views).
```
Comment (otisg) [inline docs/faq.md:34]: seperate ==> separate

**Review by otisg — CHANGES_REQUESTED** [review]
One small typo

File: docs/faq.md (hunk 1 above)
Comment (michaelhigham) [reply docs/faq.md:34]: Yes sorry I will change it now

**Comment by michaelhigham** [comment]
Fixed

**Review by radu-gheorghe — APPROVED** [review]

**Pull request merged**